.github/ISSUE_TEMPLATE/release.md

```diff
@@ -24,8 +18,16 @@ Release checklist:
   Add entries to `.mailmap` to remove duplicates.
 - [ ] Run [hoomd-benchmarks](https://github.com/glotzerlab/hoomd-benchmarks), check for performance
   regressions with the previous release, and post the tables in the release pull request.
+- [ ] Run [hoomd-validation](https://github.com/glotzerlab/hoomd-validation).
+
+All releases:
+
+- [ ] Update change log.
+  - ``git log --format=oneline --first-parent `git log -n 1 --pretty=format:%H -- CHANGELOG.rst`...``
+  - [milestone](https://github.com/glotzerlab/hoomd-blue/milestones)
+- [ ] Check readthedocs build, especially change log formatting.
```

Comment:
Did you want to remove build status and output?

Reply:
Yes, this is intentional. This was replaced by the readthedocs integration in the PR Checks section long ago.